PurchasesTests/Networking/BackendTests.swift

```diff
@@ -1055,16 +1054,13 @@ class BackendTests: XCTestCase {
         
         let currencyCode = "BFD"
         
-        let paymentMode: RCPaymentMode = .none
+        let paymentMode: ProductInfo.PaymentMode = .none
         
         var completionCalled = false
-        
-        let discount = RCPromotionalOffer()
-        discount.offerIdentifier = "offerid"
-        discount.paymentMode = .payAsYouGo
-        discount.price = 12
 
-        let productInfo: RCProductInfo = .createMockProductInfo(productIdentifier: productIdentifier,
+        let discount = PromotionalOffer(offerIdentifier: "offerid", price: 12, paymentMode: .payAsYouGo)
+
+        let productInfo: ProductInfo = .createMockProductInfo(productIdentifier: productIdentifier,
```

Comment:
indentation is slightly off for the next few lines

Reply:
✅